src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -65,9 +65,14 @@ public VisualStudioWorkspaceImpl(ExportProvider exportProvider)
             : base(
                 MefV1HostServices.Create(exportProvider))
         {
+            _threadingContext = exportProvider.GetExportedValue<IThreadingContext>();
             _textBufferCloneService = exportProvider.GetExportedValue<ITextBufferCloneService>();
             _textBufferFactoryService = exportProvider.GetExportedValue<ITextBufferFactoryService>();
             _projectionBufferFactoryService = exportProvider.GetExportedValue<IProjectionBufferFactoryService>();
+            _linkedFileUtilities = exportProvider.GetExportedValue<LinkedFileUtilities>();
+
+            _foregroundObject = new ForegroundThreadAffinitizedObject(_threadingContext);
```

Comment:
Please test `devenv /build` before we break it again.

Reply:
➡️ Added to the list for follow-up. Unless you say otherwise, I'm planning to treat this one as a "priority follow-up" item.